test/serving.bash

```diff
@@ -184,3 +161,38 @@ function upstream_knative_serving_e2e_and_conformance_tests {
       --imagetemplate "$image_template"
   fi
 }
+
+function enable_feature_flags {
+  local failed=0
+
+  for feature in "$@"; do
+    echo "Enabling feature: $feature"
+    configure_cm features "$feature":enabled || failed=1
+  done
+  # Allow settings to be picked up
+  sleep 30
+  return $failed
+}
+
+function configure_cm {
+  local failed=0
+  local cm="$1"
+  local patch=""
+  declare -A json_properties
+
+  for property in "${@:2}"; do
+    KEY="${property%%:*}"
+    VALUE="${property##*:}"
+    patch=${patch:+$patch,}"\"$KEY\": \"$VALUE\""
+    # escape in case property contains dots eg. kubernetes.pod-spec
+    j_property="$(echo "'$KEY'" | sed "s/\./\\\./g")"
+    json_properties["$j_property"]="$VALUE"
+  done
+
+  oc -n ${SYSTEM_NAMESPACE} patch knativeserving/knative-serving --type=merge --patch="{\"spec\": {\"config\": { \"$cm\": {$patch} }}}" || failed=1
```

Comment:
We use "${SERVING_NAMESPACE}" in this repo. That's also the reason why the tests failed.